nyan/value/dict.cpp

```diff
@@ -245,6 +245,14 @@ const std::unordered_set<nyan_op> &Dict::allowed_operations(const Type &with_typ
 		nyan_op::INTERSECT_ASSIGN,
 	};
 
+	if (with_type.get_primitive_type() == primitive_t::NONE) {
+		return none_ops;
```

Comment:
`None` is assigned directly to the member, it's not applied like the other values. However, we still need to list it here because assigning None is an allowed operation.